test/run-tests.sh

```diff
@@ -86,6 +87,16 @@ if [  "${CLUSTER_WIDE}" == "true" ]; then
   sleep 2
   kubectl wait --for=condition=ready pod -l control-plane=controller-manager --timeout=600s -n splunk-operator
   if [ $? -ne 0 ]; then
+    echo "kubectl get pods -n kube-system ---"
+    kubectl get pods -n kube-system
+    echo "kubectl get deployement ebs-csi-controller -n kube-system ---"
+    kubectl get deployement ebs-csi-controller -n kube-system
+    echo "kubectl describe pvc -n splunk-operator ---"
+    kubectl describe pvc -n splunk-operator
```

Comment:
Nit: Do we need these commands?

Reply:
yes, its hard to debug if we do not know what happened in failure case